Standups-StackBased/Standups-StackBased.xcodeproj/project.pbxproj

```diff
@@ -427,7 +428,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 16.2;
+				IPHONEOS_DEPLOYMENT_TARGET = 17.0;
```

Comment:
updated deployment to iOS 17 but don't have any of the `@Observable` stuff done. that will come later.